spec/controllers/reports_controller_spec.rb

```diff
@@ -2,20 +2,21 @@
 
 describe ReportsController, type: :controller do
   before(:all) do
```

Comment:
Is it possible that we might run into similar issues here because of the `before(:all)` call? Were you thinking we'd leave refactoring these with mocks to #1412?